drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1439,105 +1428,7 @@ RID TextureStorage::render_target_get_texture(RID p_render_target) {
 	RenderTarget *rt = render_target_owner.get_or_null(p_render_target);
 	ERR_FAIL_COND_V(!rt, RID());
 
-	if (rt->external.fbo == 0) {
```

Comment:
@dsnopek just a heads up though I think we already talked about this, note that I'm removing the old external texture logic as it has become defunct but am not implementing the new feature for GLES3, only for Vulkan.

We'll eventually need to implement the new approach for GLES3 as well, something best done after your multiview changes are completed (or as part thereof).

Reply:
Yep, that sounds good!

It probably makes sense to add it after the multiview PR (as opposed to being a part of it) because I'll need to make some pretty big changes to WebXR (switching to using the WebXR Layers API) in order to really test that it works, and there's already lots to do in order to get the multiview PR done.